import/index_java/src/org/vufind/index/CreatorTools.java

```diff
@@ -592,6 +593,12 @@ protected List<String> normalizeRelatorStringList(List<String> stringList)
      */
     protected String normalizeRelatorString(String string)
     {
+        for (String prefix : relatorPrefixesToStrip) {
```

Comment:
Would it make sense to do this last? Otherwise e.g. leading whitespace will cause the stripping to seemingly not work.

Reply:
Good point. I've moved the trim above the prefix stripping. I think it may make more sense to keep the lowercasing and punctuation stripping later (since otherwise we would have to apply the same operations to the prefixes themselves, and that might result in unexpected behaviors).

Reply:
Makes sense!